### DISCUSSION TITLE
Can I contribute samples for Vue Components

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#11078] [General - Components / Directives / etc]
I was wondering if I could contribute samples to the Vue Components showcased on the pages of the Quasar website?

I have a sample similar to this sample seen on the vue website that i find myself referencing every once in awhile, it would be nice to have on the Quasar website. I can supply you the code.

https://vuetifyjs.com/en/components/treeview/#load-children

**Comment by JokerMartini** [comment]
I also have a small 10 lines of code snippet to showcase a commonly asked sample I've seen online. I can post here.

**Reply by JokerMartini** [reply]
Regards to Flex layout

**Reply by JokerMartini** [reply]
For example anyone making grid layouts for galleries this snippet has always been very useful for me. It'd be great to see you add to the Quasar docs page in the Flex/ Layout section https://codepen.io/jokermartini/pen/XWaZWom

**Comment by suleimanas** [comment]
https://quasar.dev/contribution-guide/contribution-guide
